### ISSUE TITLE
Passkey Popup Spam on New Service (Linkedin)

### ISSUE THREAD

**Issue by mtt59** [ferdium/ferdium-app#1801]
### Avoid duplicates

- [X] I agree to follow the [Code of Conduct](https://github.com/ferdium/ferdium-app/blob/HEAD/CODE_OF_CONDUCT.md) that this project adheres to.
- [X] I have searched the [issues tracker](https://github.com/ferdium/ferdium-app/issues?q=is%3Aopen+is%3Aissue+label%3A%22bug+%3Abeetle%3A%22) for a bug report similar to mine, in vain

### Ferdium Version

6.7.4

### What Operating System are you using?

Windows

### Operating System Version

Windows 11 23H2 Build#22631.3672

### What arch are you using?

x64

### Last Known Working Ferdium version

n/a

### Expected Behavior

Login page for the linkedin service, with a manual or single prompt popup for a passkey

### Actual Behavior

Constant popups for logging in with a passkey when adding linkedin service.
Clicking cancel makes the window go away only for it to come back about a second later. This cycle does not end after 10+ rounds.
This popup happens even when the linkedin service is not in focus. See attached screenshots

### Steps to reproduce

1. Go to add services & add a default linkedin service.
2. Click on the new service to load the login webpage
3. popup happens

### Debug link

https://debug.ferdium.org/140709c1-94bf-47b0-92a5-20070fa5f112

### Screenshots

![Screenshot 2024-06-04 142431](https://github.com/ferdium/ferdium-app/assets/55414564/a874c9f1-bb9d-4547-b437-4b554400a38c)
![Screenshot 2024-06-04 143227](https://github.com/ferdium/ferdium-app/assets/55414564/8d327740-2e3d-44f0-a73b-2b112ef8a306)

### Additional information

I have only encountered this problem on linked in, and I don't know if this happens on other services since the passkey prompt is a Windows 11 OS feature.

In other words: I don't know if this is a problem with: 
- the way Linkedin javascript prompts for the passkey,
- the way Ferdium handles passkey requests
- the way Windows OS handles passkey requests

Edit:
My workaround is to quickly paste in login info in between passkey prompts like a videogame quicktime event. The passkey prompt popups cease after the first login page; When the one-time email code page opens.

**Comment by SpecialAro** [comment]
Hello @mtt59, thank you for logging this issue!

Does this happen in a regular web browser also?

The recipe url is pointing to:
https://www.linkedin.com/messaging

**Comment by SpecialAro** [comment]
Just noticed this can be, in fact, an Electron "bug"

https://www.reddit.com/r/electronjs/comments/1b4ke3g/linkedin_authentication_issues_passkey_popup/

If someone cares to pick this up, the OP updated the post with a workaround. Otherwise, once I have time I can try to make a fix

**Comment by audricd** [comment]
I can confirm this happens too, on windows.
On linux there is another problem (but I will raise another ticket for that)

**Comment by rodney-zhu** [comment]
This is driving me nuts on Windows 11. Google is the one asking me constantly though for my key: 
![image](https://github.com/user-attachments/assets/450d0315-6cca-4a31-80c8-a138c9be2656)
![image](https://github.com/user-attachments/assets/3ab63589-bf3d-4465-96ce-a93c47c48609)

**Comment by awilliansd** [comment]
Someone else?

**Comment by chicknlil** [comment]
Was this ever resolved? I have this issue with Google AND Paypal within Ferdium.

**Comment by gabrielaquilano** [comment]
> Was this ever resolved? I have this issue with Google AND Paypal within Ferdium.

it was not ; still happening, latest version.

**Comment by Yampanis** [comment]
This is happening to me today.  Seems related to LinkedIn.

**Comment by rgarcia7** [comment]
Happening to me today also
Cleared cache, but still same issue

![Image](https://github.com/user-attachments/assets/20486a91-39e6-4696-839d-fd443a0844f3)

```
Version: 7.0.1
Server: No Server
Electron: 34.4.1
Chrome: 132.0.6834.210
Node.js: 20.18.3
Platform: Windows 10
Arch: x64
Build date: Mon Apr 07 2025 10:21:53 GMT-0600 (Central Standard Time)
Git SHA: 0e27e4da
Git branch: release
```

**Comment by thepra** [comment]
I'm on Windows 10 and I have the linkedin spam too

**Comment by T4ke-Github** [comment]
Problem still persist and blocks the whole App if you have the LinkedIn application active.

Version: 7.1.0
Server: Ferdium Server
Electron: 35.4.0
Chrome: 134.0.6998.205
Node.js: 22.15.0
Platform: Windows 11
Arch: x64
Build date: Sun May 25 2025 17:47:00 GMT+0200 (Central European Summer Time)
Git SHA: c5030181
Git branch: release

**Comment by karserasl** [comment]
can we get a workaround atleast to not lock the whole app?

**Comment by audricd** [comment]
> can we get a workaround atleast to not lock the whole app?

you got to be real fast and disable the service

**Comment by DoYouKnowMyNamePlz** [comment]
Still an issue and really annoying.

**Comment by DoYouKnowMyNamePlz** [comment]
Hmm weird. I removed Facebook from my services and now I am not getting the popup. Can anyone else test and verify this?

**Comment by Shpigorx** [comment]
Request appears from Facebook Meta. Disabling the META service and then deleting it did not change anything. There are also active Instagram tabs. Popup still appears either rarely or constantly from META. Behavior is incomprehensible.

Checked that 1 of IG services was not logged in. It seems to have solved the problem.
